src/orchard/inspect.clj

```diff
@@ -77,23 +74,35 @@
       (pop (pop path)) ; pop twice to remove <(find :some-key) key>
       (pop path))))
 
-(defn clear
-  "Clear an inspector's state."
-  [inspector]
-  (merge (reset-index inspector)
-         {:value nil, :stack [], :path [], :pages-stack [],
-          :current-page 0, :rendered [], :indentation 0}))
+(def ^:private default-inspector-config
+  "Default configuration values for the inspector."
+  {:page-size        32
```

Comment:
In light of our previous conversation we might also add a couple of short comments explaining how the defaults were chosen.

Reply:
I will add comments to those where the default isn't completely arbitrary.